packages/ckeditor5-ui/src/dialog/dialogactionsview.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/dialog/dialogactionsview
+ */
+
+import {
+	FocusTracker,
+	KeystrokeHandler,
+	type Locale,
+	type CollectionChangeEvent
+} from '@ckeditor/ckeditor5-utils';
+import type Button from '../button/button';
+import ButtonView from '../button/buttonview';
+import View from '../view';
+import ViewCollection from '../viewcollection';
+import FocusCycler from '../focuscycler';
+
+/**
+ * TODO
+ */
+export default class DialogActionsView extends View {
```

Comment:
Please check which properties of this class must be public. E.g. I am not sure about `keystrokes`. But maybe this is required by some type/interface.